bee/src/experimental/java/com/apisense/bee/EventObserverApplication.java

```diff
@@ -0,0 +1,27 @@
+package com.apisense.bee;
+
+import com.apisense.bee.utils.accessibilitySting.AccessibilitySting;
+
+import java.util.Collections;
+import java.util.List;
+
+import io.apisense.sdk.APISENSE;
+import io.apisense.sdk.core.sting.InjectedStingPackage;
+import io.apisense.sdk.core.sting.StingComponent;
+import io.apisense.sting.lib.Sting;
+
+/**
+ * Created by Mohammad Naseri
+ */
+
+public class EventObserverApplication extends BeeApplication {
```

Comment:
We will use this flavor to test another sting at the same time, can you change the name of this class to be more generic? I propose to call it `BeeExperimentalApp`.